hbw/production/ml_inputs.py

```diff
@@ -309,3 +312,184 @@ def dl_ml_inputs_init(self: Producer) -> None:
 
     # add variable instances to config
     add_dl_ml_variables(self.config_inst)
+
+
+@producer(
+    uses={
+        category_ids, event_weights,
+        prepare_objects,
+        "HbbJet.msoftdrop", "HbbJet.deepTagMD_HbbvsQCD",
+        "Jet.btagDeepFlavB", "Bjet.btagDeepFlavB", "Lightjet.btagDeepFlavB",
+    } | four_vec(
+        {"Electron", "Muon", "MET", "Jet", "Bjet", "Lightjet", "HbbJet", "VBFJet"},
+    ),
+    produces={
+        category_ids, event_weights,
+        # other produced columns set in the init function
+    },
+)
+def sl_res_ml_inputs(self: Producer, events: ak.Array, **kwargs) -> ak.Array:
```

Comment:
Maybe it's a better idea to combine `sl_res_ml_inputs` and `ml_inputs` into one producer since we can probably both profit from similar variables.